### PR TITLE
Digital checkout optimisations

### PR DESCRIPTION
**Pull request by damien-shaw-guardian**
## Why are you doing this?
This PR will hopefully in improve conversions on the paypal payments

[**Trello Card**](https://trello.com/c/pufad5wP/2295-paypal-optimisations)

## Changes

* Add payment logo to payment radio buttons
* Moved payment button above Money Back Guarantee text

## Screenshots
Screen Shot 2019-03-14 at 17 38 37

### CODE REVIEW

File: support-frontend/assets/pages/digital-subscription-checkout/components/addressForm.jsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+import { withError } from 'hocs/withError';
+import { asControlled } from 'hocs/asControlled';
+import { canShow } from 'hocs/canShow';
+
+// components
+import { Input } from 'components/forms/input';
+import { Select } from 'components/forms/select';
+import { sortedOptions } from 'components/forms/customFields/sortedOptions';
+
+const InputWithLabel = withLabel(Input);
```
Comment (rupertbates) [inline support-frontend/assets/pages/digital-subscription-checkout/components/addressForm.jsx]: Can you extract these to a new module in `components/subscriptionsCheckouts` and import them into this file and the checkout form? It would be good to use them in the paper checkout form as well because they are also declared in there, but up to you if you want to leave that for another PR

File: support-frontend/assets/pages/digital-subscription-checkout/components/checkoutForm.jsx [hunk 2]
```diff
@@ -7,27 +7,24 @@ import { connect } from 'react-redux';
...
 import { RadioInput } from 'components/forms/customFields/radioInput';
 import { withLabel } from 'hocs/withLabel';
 import { withError } from 'hocs/withError';
 import { asControlled } from 'hocs/asControlled';
-import { canShow } from 'hocs/canShow';
 import Form, { FormSection } from 'components/checkoutForm/checkoutForm';
 import Layout from 'components/subscriptionCheckouts/layout';
 import GeneralErrorMessage from 'components/generalErrorMessage/generalErrorMessage';
 import Content from 'components/content/content';
+import AddressFrom from './addressForm';
```
Comment (rupertbates) [inline support-frontend/assets/pages/digital-subscription-checkout/components/checkoutForm.jsx]: Typo, should be `AddressForm`

File: support-frontend/assets/pages/digital-subscription-checkout/components/checkoutForm.jsx [hunk 3]
```diff
@@ -160,6 +143,15 @@ function CheckoutForm(props: PropTypes) {
...
     />) : '';
 
+  const isPayPalEnabled = (optimizeExperiments: OptimizeExperiments) => {
+    const PayPalExperimentId = '8IebFnX-SbKRxhlj_tGp-w';
+    const enabledByTest = optimizeExperiments.find(exp => exp.id === PayPalExperimentId && exp.variant === '1');
+    const enabledByQueryString = getQueryParameter('payPal') === 'true';
+    return enabledByTest || enabledByQueryString;
+  };
+
+  const payPalEnabled = isPayPalEnabled(props.optimizeExperiments);
```
Comment (rupertbates) [inline support-frontend/assets/pages/digital-subscription-checkout/components/checkoutForm.jsx]: Would be good to pass `payPalEnabled` into AddressForm instead of `optimizeExperiments` as we've already worked it out here.

File: support-frontend/assets/pages/digital-subscription-checkout/components/addressForm.jsx (hunk 1 above)
Comment (walaura) [reply support-frontend/assets/pages/digital-subscription-checkout/components/addressForm.jsx]: I'm not sure if doing so kinda goes against the spirit of the HoCs? i like having them defined inline because it makes it easy to know exactly what they are, if we unify them in one file we could as well export a single input component that takes labels and errors as optional props?

File: support-frontend/assets/components/subscriptionCheckouts/paymentMethodSelector.jsx.orig [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+<<<<<<< HEAD
+const isPayPalEnabled = (optimizeExperiments: OptimizeExperiments) => {
+  const PayPalExperimentId = 'ol9MppFXRle6XJwSRj3psA';
+  const enabledByTest = optimizeExperiments.find(exp => exp.id === PayPalExperimentId && exp.variant === '1');
+  const enabledByQueryString = getQueryParameter('payPal') === 'true';
+  return enabledByTest || enabledByQueryString;
+};
+
+=======
+>>>>>>> refactor: added paypal props to PaymentMethodSelector
```
Comment (johnduffell) [inline support-frontend/assets/components/subscriptionCheckouts/paymentMethodSelector.jsx.orig]: What's this? the whole .orig file is surely not needed?
